src/main/java/com/amazon/redshift/plugin/httpserver/Server.java

```diff
@@ -90,7 +90,10 @@ public Server(int port, RequestHandler handler, Duration waitTime,
         this.m_ipAddress = InetAddress.getLoopbackAddress();
         this.m_socketFactory = ServerSocketFactory.getDefault();
         this.m_defaultSocketConfig = SocketConfig.custom()
-            .setBacklogSize(1)
+            // The backlog size needs to be at least 2 to avoid random `ERR_SOCKET_NOT_CONNECTED`
+            // errors in Google Chrome on macOS. It's caused by Chrome's "Preload pages" feature,
+            // which is enabled by default (located here: `chrome://settings/preloading`).
+            .setBacklogSize(2)

```

Comment:
@jimmyatwork are there any performance impacts to setting backlog size to 2 here?